packages/Input/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [0.2.23] - 2020-04-29
+
+### Added
+
+- Simplified uncontrolled api removing state for value [@tristanjasper](https://github.com/tristanjasper).
```

Comment:
What's this CHANGELOG business, are we doing this now? Arent git commits the changelog?

Reply:
I believe we _are_ doing this now for changes we make, think it was discussed in a workshop a while ago..